app/Http/Controllers/PlayerController.php

```diff
@@ -13,9 +14,32 @@ class PlayerController extends Controller
      */
     public function index()
     {
-        // TODO show ALL players (route '/allespieler'; view 'players.index')
+        // show ALL players (route '/allespieler'; view 'players.index')
         $players = \App\Player::all();
```

Comment:
```suggestion
        $players = Player::all();
```
You dont need the full namespace due to your `use App\Player;` at the beginning of the file.